packages/manager/modules/bm-server-components/src/ftp-backup-storage/delete/dedicated-server-ftp-backup-delete.controller.js

```diff
@@ -0,0 +1,43 @@
+import { FTP_BACKUP_STORAGE_ALERT } from '../ftp-backup.constants';
+
+export default class DeleteFtpBackupCtrl {
+  constructor($stateParams, $translate, Server, Alerter) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.Server = Server;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.ftpBackup = this.ipbackup;
```

Comment:
What's the goal here ? If you don't make a copy, it's useless.
Moreover, it hides the use of `ipbackup` in the component